packages/types-dev/objects.d.ts

```diff
@@ -582,6 +582,24 @@ declare global {
             custom?: undefined;
         }
 
+        /**
+         * ioBroker has built-in protection for specific attributes of objects. If this protection is installed in the object, then the protected attributes of object cannot be changed by the user without valid password.
+         * To protect the properties from change, the special attribute "nonEdit" must be added to the object. This attribute contains the password, which is required to change the object.
+         * If object does not have "nonEdit" attribute, so the hash will be saved into "nonEdit.passHash". After that if someone will change the object, he must provide the password in "nonEdit.password".
+         * If the password is correct, the object attributes will be updated. If the password is wrong, the object will not be changed.
+         * Note, that all properties outside "nonEdit" can be updated without providing the password. Furthermore, do not confuse e.g. "nonEdit.common" with "obj.common" they are not linked in any way.
```

Comment:
AFAIK `Furthermore, do not confuse e.g. "nonEdit.common" with "obj.common" they are not linked in any way.` this should be correct code-wise. Please correct me if I am wrong @GermanBluefox